docs/docs/adrs/adr-017-allowing-inactive-validators.md

```diff
@@ -0,0 +1,111 @@
+* Rewards: Validators that are not part of the active set on the provider chain can still receive rewards on the consumer chain, but they *do not* receive rewards from the provider chain. This change is
+achieved without further changes to staking or reward distributions, because similar to downtime, rewards are based on the consensus validator set.
+
+## Risk Mitigations
+
+To mitigate risks from validators with little stake, we introduce a minimum stake requirement for validators to be able to validate on consumer chains, which can be set by each consumer chain independently, with a default value set by the provider chain.
+
+Additional risk mitigations are to increase the active set size slowly, and to monitor the effects on the network closely. For the first iteration, we propose to increase the active set size to 200 validators (while keeping the consensus validators to 180), thus letting the 20 validators with the most stake outside of the active set validate on consumer chains.
+
+## Consequences
+
+### Positive
+
+* Validators outside of the active set can validate on consumer chains without having an impact on the consensus engine of the provider chain
+* Consumer chains can have a much larger validator set than the provider chain if they prefer this e.g. for decentralization reasons
+
+### Negative
+
+Allowing validators from the inactive set brings with it some additional risks.
+In general, consumer chains will now face some of the problems also faced by standalone chains. It’s reasonable to assume that the validator set on the hub has a minimal amount of operational quality due to being battle tested and decentralized, and consumer chains with validators from outside the hub active set cannot rely on this as much anymore.
+
+
+#### Sybil attacks
+
+With the restricted size of the active set today, it’s clear that the set is at least minimally competitive and it is not trivial to spin up multiple nodes as a validator.
+
+When we make the “potential validator set” much larger, we should assume that it becomes much less competitive to be part of that set, and thus trivial for single entities to control many of those validators.
+
+#### Reputational damage is not a deterrent
+
+For validators in the active set, we typically assume that if they would misbehave, they pay a large reputational cost. This represents delegators deciding to switch validators (potentially even on chains other than the one the misbehaviour happened on), and loss of credibility in the ecosystem. With the much larger active set, it seems prudent to assume that reputational damage is not a deterrent for many validators. They might only have minimal amounts of delegated stake and control most of it themselves, so they might not be deterred from performing actions that would usually bring reputational damage.
+
+#### Additional negative consequences
+* Validators outside of the active set become bonded, even if they are not validating on any consumer chains
```

Comment:
What is it negative that validators are bonded?
More general question on this: Are there any side-effects by having more validators bonded? E.g., other `cosmos-sdk` modules that would operate differently when having more bonded validators?

Reply:
It's negative because it means that those validators need to go through unbonding before they get their tokens back (but it appears this is essentially already how it works, so maybe I should take that out!

Yeah, for gov proposals, those validators now would get to vote. (I mean, the validators would already now get to vote; but they would get to vote with delegated stake). I think this is probably the way it should be, too, because if those validators have delegators, we should assume they represent their delegators exactly like active set validators represent their delegators

Reply:
Although I do still think this is a negative - it's ok if a validator is bonded who is validating on at least some consumer chain, but having a validator that is bonded who is not validating anything is a bit strange to me

Reply:
I'll put it as neutral for now